src/index.js

```diff
@@ -0,0 +1,6 @@
+exports.say = function(name){
```

Comment:
Here in Cubbit we refer to the Allman indentation style (https://en.wikipedia.org/wiki/Indentation_style#Allman_style) so your code changes in:
```
exports.say = function(name)
{
    // code here
}
```